gulpfile.js

```diff
@@ -49,3 +49,12 @@ gulp.task('git-check', function(done) {
   }
   done();
 });
+
+gulp.task('inject-css', function(){
```

Comment:
I appreciate the intermediate variables rather than putting the entire return into one statement as I've seen in many examples.